variables.tf

```diff
@@ -42,15 +42,9 @@ variable "name" {
   default     = ""
 }
 
-variable "nat_ip_allocate_option" {
```

Comment:
FYI: I think removing this variable will make it a breaking change (if present if module call, `nat_ip_allocate_option` will need to be removed.)

Reply:
You are correct, removing the nat_ip_allocate_option variable will be a breaking change. What do you suggest should we keep it? I hope they have versioned their module calls.